letsencrypt/cli.py

```diff
@@ -636,8 +638,12 @@ def preprocess_args(self, args):
                 self.verb = token
                 return reordered
 
-        self.verb = "run"
-        return args + ["run"]
+        if "--standalone" in args and "--installer" not in args and "-i" not in args:
```

Comment:
It is my opinion that this is a bad decision. How much gain does the user really get here? The three cases I see are:

1. No installers are available. This is the most likely scenario given the current state of installers. This change saves them a quickly failed run telling them to run `auth`.
2. An installer is available and a webserver is running on the desired port. With or without this change, they get a message saying the port is bound.
3. An installer is available and the port is free. Without this change, they get the full functionality provided by Let's Encrypt of automatically installing the cert and configuring their installation. With this change, the user gets reduced functionality.

So the only benefit I see in case 1, which is hardly worth discussing in my opinion.

Also, what about `-a standalone`and `--authenticator standalone`? Why don't those have special behavior?